### PR TITLE
Feature Get targets

### PR DESCRIPTION
**Pull request by lnemmer**
This pr adds get targets feature and create target hot fixes.

**Screenshots**

screen shot 2018-10-08 at 3 39 10 pm
screen shot 2018-10-08 at 3 39 53 pm

### CODE REVIEW

File: src/components/home/Map.js [hunk 1]
```diff
@@ -80,18 +96,30 @@ class Map extends Component {
             <TargetMarker
               lat={currentLocation.lat}
               lng={currentLocation.lng}
-              target={currentLocationImg}
               radius={currentLocationRatio}
             />
           }
-          {targetPointerLocation &&
+          {(showMarker) &&
```
Comment (aguscha333) [inline src/components/home/Map.js]: no parenthesis required

File: src/components/common/TargetMarker.js [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+      {/* divs in lines below are necessary */}
+      {(targetImgClass === 'topic-pointer' &&
+      <div>
+        <div className="target-perimeter" style={perimeter} />
+        <div className="yellow-circle">
+          <img className={targetImgClass} src={img} alt="Location" />
+        </div>
+      </div>
+      ) ||
+      <div>
```
Comment (missx) [inline src/components/common/TargetMarker.js]: In these cases you can also use ternary operators like so: ```{targetImgClass === 'topic-pointer' ? <div>bla bla</div> : <div>ble ble</div>}```

File: src/components/home/Map.js [hunk 3]
```diff
@@ -56,15 +65,22 @@ class Map extends Component {
...
   render() {
-    const { currentLocation, showCurrentLocation, currentLocationRatio, targetPointerLocation } = this.state;
-    const { lat: latTargetMarker, lng: lngTargetMarker, radius: radiusTargetMarker } = targetPointerLocation;
+    const { currentLocation, showCurrentLocation, currentLocationRatio, targetPointerLocation: { lat, lng, radius, showMarker }, targetsQty } = this.state;
+    const { targetList } = this.props;
+
+    // If user clicks in map, a marker pointer (in memory) will be shown. In case user creates a target, then the pointer
+    // has to disappear and appears the target with its icon.
+    // If targetList length value changes (in other words, a new target is created) then I have to
+    // set show value to false (I don't want to show memory marker because the target is now in targetList), and I have to modify targetQty to current targetList length
```
Comment (missx) [inline src/components/home/Map.js]: Could you shorten lines from 75 to 81 where needed?

File: src/components/common/TargetMarker.js [hunk 4]
```diff
@@ -1,23 +1,40 @@
...
+import currentLocationImg from '../../assets/icons/location.png';
 
-const TargetMarker = ({ target, radius }) => {
+const TargetMarker = ({ img, radius }) => {
   const perimeter = {
     width: (radius && `${radius}px`) || '0px',
     height: (radius && `${radius}px`) || '0px'
   };
 
+  const targetImgClass = ((img === currentLocationImg) && 'target-pointer') || 'topic-pointer';
```
Comment (felipegarcia92) [inline src/components/common/TargetMarker.js]: Like Ximena said below, you can/should use the ternary operator here. Something like:
`const targetImgClass = (img === currentLocationImg) ? 'target-pointer' : 'topic-pointer'`

**Pull request merged**